packages/keyring-controller/src/KeyringController.ts

```diff
@@ -2191,15 +2211,30 @@ export class KeyringController extends BaseController<
    * Restore a serialized keyrings array.
    *
    * @param serializedKeyrings - The serialized keyrings array.
+   * @returns A promise resolving to the restored keyrings.
```

Comment:
Nit: We can omit "A promise resolving to", as it is implied by the type signature.

```suggestion
   * @returns The restored keyrings.
```